### ISSUE TITLE
plotProfile() needs more uniformity in spacing for units

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1040]
See e.g. the snapshot: the sig-the axis of the top-right panel should have spaces between the units and the `[]` brackets. I think maybe the same is true for temperature but I'm not sure. Then again, maybe N2 has *too* much space. Uniformity might be good, but the important thing is readibility.

**Comment by dankelley** [comment]
Forgot the picture...

![04_091](https://cloud.githubusercontent.com/assets/99469/17386083/8e1ad0d2-59bd-11e6-99c1-6a21b9ce3847.png)

**Comment by richardsc** [comment]
If you were looking for votes, I think I prefer the version with less space (as in T and sigma). Though maybe something halfway between the two would be ideal ... ?

**Comment by dankelley** [comment]
I've tightened things up in `develop` commit 5344171781728bbd83f39d2dfdb5989a53329b56 and a screenshot is below. I don't know how to make a fractional space, so I just went to no space. Thus we now rely on R typesetting to establish clarity e.g. preventing a trailing superscript from hitting the left side of a `]` character.  A screenshot is below.

The issue as stated in the title has been addressed, so I'll close this. (I'm not saying that discussion should be closed, of course, since we can always go back and forth on closed issues. The reason to close is that new users don't need to see this when they visit the issues site to find out whether the serious bug they just found has already been reported.)

![a](https://cloud.githubusercontent.com/assets/99469/17462165/a8a2d1c4-5c7a-11e6-8264-72f03937e2af.png)
